module/README.md

```diff
@@ -31,6 +31,39 @@ async def root():
     return {"message": "Hello World"}
 ```
 
+# Prometheus Metrics Middleware
+
+Here's how you can use the middleware to define your own custom metrics, which the otel will scrape on the `/metrics` endpoint added to your app automatically.
+
+```
+from starlette.applications import Starlette
+from prometheus_metrics_middleware import expose_metrics, create_custom_metrics, CollectorRegistry
+
+app = Starlette() / FastAPI() # your app can be either
+
+# Create a registry
+registry = CollectorRegistry()
+
+# Create custom metrics
+custom_metrics = create_custom_metrics(registry)
+
+# Add more custom metrics if needed
+def update_business_metric(request, response, duration):
+    # Update your business-specific metrics here
```

Comment:
This would be way, way more useful if it logged an example metric. Let's imagine for a moment a ranker that computes a score that it uses to rank each content item. This could log a histogram of the scores, tagged by platform.